### PR TITLE
css-box-model

### PR DESCRIPTION
**Pull request by Jason-Humphrey**
Screen Shot 2021-01-20 at 1 46 30 PM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+}
+.language {
```
Comment (codingcodymiller) [inline css-box-model/styles.css:5]: There is already a selector for `button` elements with the `language` class above. This separate rule isn't adding anything new, so this styling can just go in the rule for `button` elements with the `language` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far, dude! Please see my notes

File: css-box-model/styles.css (hunk 1 above)
Comment (Jason-Humphrey) [reply css-box-model/styles.css]: Ok, i changed it and pushed the commit.

Screen Shot 2021-01-20 at 1 55 50 PM
Screen Shot 2021-01-20 at 1 46 30 PM

**Review by codingcodymiller — APPROVED** [review]
Awesome!! 👍

**Pull request merged**